app/views/pages/cookies.html.erb

```diff
@@ -0,0 +1,161 @@
+    </p>
+
+    <h2 class="govuk-heading-l">
+      How we use cookies
+    </h2>
+
+    <h3 class="govuk-heading-m">
+      Measuring website usage (Google Analytics)
+    </h3>
+
+    <p class="govuk-body">
+      We use Google Analytics software to collect information about how you use Teachers: claim back your student loan repayments.
+      We do this to help make sure the site is meeting the needs of its users and to help us make improvements.
+    </p>
+
+    <p class="govuk-body">
+      Google Analytics stores information about:
+    </p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>the pages you visit on Teachers: claim back your student loan repayments</li>
+      <li>how long you spend on each page</li>
+      <li>how you got to the site</li>
+      <li>what you click on while you're visiting the site</li>
+    </ul>
+
+    <div class="panel panel-border-wide">
+      <p class="govuk-body">
+        We don't allow Google to use or share our analytics data.
+      </p>
+    </div>
+
+    <p class="govuk-body">
+      Google Analytics sets the following cookies:
+    </p>
+
+    <table class="govuk-table mb1">
+      <thead class="govuk-table__head">
+        <tr>
+          <th class="govuk-table__header">Name</th>
+          <th class="govuk-table__header">Purpose</th>
+          <th class="govuk-table__header">Expires</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr class="govuk-table__row">
+          <td class="govuk-table__cell">_ga</td>
+          <td class="govuk-table__cell">This helps us count how many people visit teaching-vacancies.service.gov.uk by tracking if you've visited before</td>
+          <td class="govuk-table__cell">After 2 years</td>
+          </tr>
+          <tr class="govuk-table__row">
+          <td class="govuk-table__cell">_gid</td>
+          <td class="govuk-table__cell">This helps us count how many people visit teaching-vacancies.service.gov.uk by tracking if you've visited before</td>
+          <td class="govuk-table__cell">After 24 hours</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <p class="govuk-body">
+      You can <a class="govuk-link" href="https://tools.google.com/dlpage/gaoptout">opt out of Google Analytics.</a>
+    </p>
+
+    <h3 class='govuk-heading-m'>
+      Sessions
+    </h3>
+
+    <p class="govuk-body">
+      A cookie is set to record your session activity.
+    </p>
+
+    <table class='govuk-table mb1'>
+      <thead class='govuk-table__head'>
+        <th class='govuk-table__header'>Name</th>
+        <th class='govuk-table__header'>Purpose</th>
+        <th class='govuk-table__header'>Expires</th>
+      </thead>
+      <tbody>
+        <tr class='govuk-table__row'>
+          <td class='govuk-table__cell'>SESSION COOKIE</td>
+          <td class='govuk-table__cell'>Stores session data</td>
+          <td class='govuk-table__cell'>After 1 hour</td>
```

Comment:
Actually this is [30 minutes at the moment](https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/app/controllers/claims_controller.rb#L2).